checkov/kubernetes/runner.py

```diff
@@ -195,7 +195,7 @@ def mutateKubernetesGraphResults(
                     check_name=check.name,
                     check_result=check_result,
                     code_block=entity_context.get("code_lines"),
-                    file_path=entity_file_path,
+                    file_path=f"/{os.path.relpath(entity_file_abs_path, root_folder)}",
```

Comment:
Can the expression be refactored into a function? it can easily be missed if it's inline